tabs-ex/src/components/Tab.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function Tab(props) {
+  let className = 'tab-list-item';
+
+  if (props.activeTab === props.label) {
+    className += ' tab-list-active';
+  }
+
+  return (
+    <li className={className} onClick={() => props.onClick(props.label) }>
```

Comment:
Anonymous functions are not used when passing props to a component. Research why and alternates.